resources/assets/less/bem/bootstrap/modal-header.less

```diff
@@ -0,0 +1,81 @@
+/**
+*    Copyright 2015 ppy Pty. Ltd.
+*
+*    This file is part of osu!web. osu!web is distributed with the hope of
+*    attracting more community contributions to the core ecosystem of osu!.
+*
+*    osu!web is free software: you can redistribute it and/or modify
+*    it under the terms of the Affero GNU General Public License as published by
+*    the Free Software Foundation, either version 3 of the License, or
+*    (at your option) any later version.
+*
+*    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+*    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+*    See the GNU Affero General Public License for more details.
+*
+*    You should have received a copy of the GNU Affero General Public License
+*    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+.modal-header {
+	border-radius: 5px 5px 0 0;
+	border-bottom: 0;
+
+	// Elements
+	&__badges {
+		display: flex;
+		position: absolute;
+		top: -20px;
+		left: 10px;
+	}
+
+	&__dimmer {
+		background: -moz-linear-gradient(top, rgba(0,0,0,0.5) 0%, rgba(0,0,0,0.75) 100%); /* FF3.6+ */
```

Comment:
my eyes kind of harmed +_+ also changing it is going to be difficult.

And please use `fade(#abc, x%)` instead of rgba.